packages/manager/apps/pci-gateway/src/pages/add/AddGateway.page.tsx

```diff
@@ -25,6 +23,10 @@ import {
 import { ODS_ICON_NAME, ODS_ICON_SIZE } from '@ovhcloud/ods-components';
 import { ShellContext } from '@ovh-ux/manager-react-shell-client';
 import { OdsHTMLAnchorElementTarget } from '@ovhcloud/ods-common-core';
+import {
+  isDiscoveryProject,
```

Comment:
`isDiscoveryProject` seems unused in this file

Reply:
it's used, just check the entire file